mufasa/guess_refine.py

```diff
@@ -258,8 +258,9 @@ def refine_guess(map, min=None, max=None, mask=None, disksize=1):
     if max is not None:
         map[map>max] = np.nan
 
-    if np.sum(np.isfinite(map)) == 0:
-        # if there are no valid pixel in the guesses, set it to one of the limits or zero
+    # if np.sum(np.isfinite(map)) == 0:
+    if np.sum(np.isfinite(map)) <2:
```

Comment:
This change could be reverted, I was running into issues where this was being triggered when it shouldn't, but I never really looked into it. Not sure if it would still happen

Reply:
For simplicity, I'm going to approve the merge and add the suggested change on my end. I also think I know what issue you may be running into now, and I'll open an issue ticket accordingly.